src/Runtime/PyOMCompileExecutionSession.hpp

```diff
@@ -47,7 +47,7 @@ PYBIND11_MODULE(PyCompileAndRuntime, m) {
       m, "OMCompileExecutionSession")
       .def(py::init<const std::string &, const std::string &, const bool,
                const bool>(),
-          py::arg("input_model_name"), py::arg("flags"),
+          py::arg("input_model_name"), py::arg("flags") = "",
```

Comment:
@AlexandreEichenberger, I set a default value for `flags`. Let me know if it is ok or not. Thanks!

Reply:
I restored this change. So I used an empty string for flags in the python script for `make check-multiple-models` instead of changing it here.